docs/styling/responsive.md

```diff
@@ -17,14 +17,94 @@ rx.text(
 
 The text will change color based on your screen size. If you are on desktop, try changing the size of your browser window to see the color change.
 
+You can also use `rx.breakpoints`. Each size maps to a corresponding key, the value of each will be applied when the screen size is greater than or equal to the named breakpoint.
+
+```python demo
+rx.text(
+    "Hello World",
+    color=rx.breakpoints(
+        initial="orange",
+        sm="purple",
+        lg="green",
+    ),
+)
+```
+
+You can also specify custom values per component using a dictionary instead of named parameters.
```

Comment:
```suggestion
Custom breakpoints in CSS units can be mapped to values per component using a dictionary instead of named parameters.
```

Personally I think avoiding the "You can ..." phrasing makes the documentation sound more authoritatize and attributes the capability/behavior to the feature rather than the user. Sometimes the "You can ..." phrasing makes sense, and is used in other pages of the docs, but generally should be avoided.

Reply:
That makes sense, will try to stick to that in the future!